.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [published]
```

Comment:
This [will also](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=published#release) trigger for draft releases:
> A release, pre-release, or draft of a release was published.

Is this what was expected?

Reply:
> This [will also](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=published#release) trigger for draft releases:
> 
> > A release, pre-release, or draft of a release was published.
> 
> Is this what was expected?

Yes.  There are several projects I use that do pre-releases all the way into NPM.  `npm info eslint versions` for an example.